### PR TITLE
Templates now have a download page for getting ZIP, viewing demo online, accessing GitHub

### PR DESCRIPTION
**Pull request by alankent**
Instead of clicking on templates (via menu Documentation / Templates) to download a ZIP file, you now get taken to a download page with a "View on GitHub", "View Demo", and "Download ZIP" link. This allows you to see the demo and source code without having to download the ZIP file and extract it (which is not possible from an iPad for example).

Also includes some fixes for some of the templates as path names were incorrect. (There are still some missing image files.)

Note: there is some inconsistency in the template names and ZIP file names (e.g. bikeshop is in e-commerce.zip). That could be improved in a later commit if deemed worthwhile, which would help remove some of the redundancy in the YAML files.

This is phase one of the "AMP Start 2.0" effort to move AMP Start content over to amp.dev (including reworking the examples to be component based). This commit is a first step. These download pages are going to be extended to include links to all pages in each demo with component / partials information and UX advice.

### CODE REVIEW

**Comment by alankent** [comment]
Commited with wrong email address. Going to resubmit.

**Comment by alankent** [comment]
Well, I followed the instructions I found to change the email address of past commits... but it seems to have created new commits without fixing the old commits, so it's still blocked on the CLA.

File: frontend/templates/views/detail/template-detail.j2 [hunk 1]
```diff
@@ -2,20 +2,134 @@
...
+        {% endif %}
+        {% if doc.teaser.github_url or doc.teaser.template_github_home %}
+          {% if doc.teaser.github_url %}
+            {% set github_url = doc.teaser.github_url %}
+          {% else %}
+            {% set github_url = 'https://github.com/ampproject/amp.dev/tree/future/templates/' + doc.teaser.template_github_home %}
+          {% endif %}
+          <a href="{{ github_url }}" rel="noopener" class="ap-m-lnk">
+            <svg class="ap-a-ico ap-m-lnk-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#external"></use></svg>
+            <span class="ap-m-lnk-text">{{ _('Go to GitHub') }}</span>
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply frontend/templates/views/detail/template-detail.j2]: ping

**Comment by sebastianbenz** [comment]
I've noticed that for most templates there are no screenshots. Is that intentional?

**Comment by sebastianbenz** [comment]
friendly ping

**Comment by matthiasrohmer** [comment]
Replaced by #3178 and #3181.

**Pull request closed without merging**